prometheus-to-sd/Makefile

```diff
@@ -16,13 +16,13 @@ all: build
 
 ENVVAR = GOOS=linux GOARCH=amd64 CGO_ENABLED=0
 PREFIX = gcr.io/google-containers
-TAG = v0.2.6
+TAG = v0.2.7
```

Comment:
This line looks like a mistake, you shouldn't bump prom-to-sd version in this PR